### DISCUSSION TITLE
Does llama.cpp support gemma 2 9b to convert to gguf?

### DISCUSSION THREAD

**Discussion by RakshitAralimatti** [ggml-org/llama.cpp#8318] [General]
Can I convert gemma 2 7b into gguf format?

**Comment by ggerganov** [comment]
Yes:

- clone https://huggingface.co/google/gemma-2-9b
- convert with `python3 convert_hf_to_gguf.py /path/to/gemma-2-9b/`

**Comment by RakshitAralimatti** [comment]
@ggerganov  Thanks for your response. I will try it !!
